File: app/src/main/java/com/wafflestudio/snutt2/adapter/IntroPagerAdapter.java
package com.wafflestudio.snutt2.adapter;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

import com.wafflestudio.snutt2.ui.IntroFragment;

/**
 * Created by makesource on 2017. 6. 23..
 */

public class IntroPagerAdapter extends FragmentPagerAdapter {

    public IntroPagerAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        // getItem is called to instantiate the fragment for the given page.
        // Return a PlaceholderFragment (defined as a static inner class below).
        return IntroFragment.newInstance(position);
    }

    @Override
    public int getCount() {
        // Show 3 total pages.
        return 3;
    }

}

File: app/src/main/java/com/wafflestudio/snutt2/adapter/IntroPagerAdapter.kt
package com.wafflestudio.snutt2.adapter

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter
import com.wafflestudio.snutt2.ui.IntroFragment

/**
 * Created by makesource on 2017. 6. 23..
 */
class IntroPagerAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {
    override fun getItem(position: Int): Fragment {
        // getItem is called to instantiate the fragment for the given page.
        // Return a PlaceholderFragment (defined as a static inner class below).
        return IntroFragment.newInstance(position)
    }

    override fun getCount(): Int {
        // Show 3 total pages.
        return 3
    }
}
